src/Mod/TechDraw/Templates/zh_CN/landscape/A0_Landscape_CN_CLIP.svg

```diff
@@ -275,7 +275,7 @@ e">
         <polygon points="598,828 606,828 602,834" />
 	</g>
 
-	<g id="bunding hole">
+	<g id="binding hole">

```

Comment:
Click the `<>` button in the top right of this window to view the SVG code. Here is the fix I made.

Reply:
@WandererFan can you confirm this fix is kosher?